New Entry: General Font Awesome (#849)
Codecademy/docs: content/general/concepts/font-awesome/terms/fa-pull/fa-pull.md

---
Title: 'fa-pull'

> Description: 'Used to put the icon to the left or right of the container, allowing contents to flow around it.'

Reviewer: ```suggestion
Description: 'Puts the icon to the left- or right-side of its container and allows contents to flow around it.'
```

Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'World Wide Web'
  - 'Fonts'
CatalogContent:
  - 'learn-css'
  - 'paths/front-end-engineer-career-path'
---

> The **`fa-pull`** classes are used to put the icon to the left or right of the container, allowing contents to flow around it. The `fa-pull-right` class places the icon on the right and the `fa-pull-left` class puts the icon on the left.

Reviewer: ```suggestion
The **`fa-pull`** classes are used to put the icon to the left- (`fa-pull-left`) or right-side (`fa-pull-right`) of its container, allowing contents to flow around it.
```
